ci/tests/test_00_unit/test_05_index_ci/indexcibase.py

```diff
@@ -0,0 +1,29 @@
+from os import path
+
+from ci.tests.base import BaseTestCase
+import ci.container_index.lib.state as index_ci_state
+
+SETUP_PACKAGES = False
+DUMMY_INDEX_FILE = "./test.yaml"
+
+
+class IndexCIBase(BaseTestCase):
+
+    node = "controller"
+
+    @staticmethod
+    def _print_init_msg(msg):
+        count = len(msg)
+        print("\n")
+        print("=" * count)
+        print(msg)
+        print("=" * count)
```

Comment:
unnecessary 4 lines of print statements which could have been reduced to
`\n{}\n{}\n{}`

Reply:
Removed